### DISCUSSION TITLE
How do black beans affect your gout?

### DISCUSSION THREAD

**Discussion by kct2020** [kct2020/goutpal-com-skeleventy#103] [Polls]
I'm reviewing readers concerns about black beans. So first, I'm asking for your biggest black bean and gout concern. Then, I'll replace the current pages with a new black bean & gout FAQ.
Please read the list of common concerns. Then select the one that means most to you. Or add other concerns as comments.

**Comment by buscemrx** [comment]
If I eat black beans I have a mild gout flare up so I have to stay away from black beans
